lib/streamlit/ScriptRunner.py

```diff
@@ -329,6 +333,7 @@ def _run_script(self, rerun_data):
             # ScriptRunner.
 
         finally:
+            media_file_manager.reset_files_for_session()
```

Comment:
I thought we decided this wasn't the right location for this. Maybe you forgot to push changes?

Also: when I try this locally, the location where this line works correctly is at the top of `_run_script`, like this:

```
        # Reset DeltaGenerators and widgets.
        media_file_manager.reset_files_for_session()
        get_report_ctx().reset()
```

Reply:
We haven't figured out the right place to put this yet.  So no changes have been pushed.

Reply:
Oh wait, this causes a (mild) memory leak, doesn't it?